### DISCUSSION TITLE
[讨论-01]pyqt开发使用的编辑器

### DISCUSSION THREAD

**Discussion by 625781186** [PyQt5/PyQt#177] [Q&A]
评估标准:
1.硬件消耗;
2.智能提示;
...

**Comment by 625781186** [comment]
群主使用 : [https://pyqt5.com/suggesteditor.html](https://pyqt5.com/suggesteditor.html)

**Comment by 625781186** [comment]
本人使用 : pycharm ; 
通过提取 eric6的源码 
![image](https://user-images.githubusercontent.com/30521429/60707209-c812af80-9f3d-11e9-934f-889fdf9d8887.png)

**Comment by 625781186** [comment]
推荐新手 : 
1. 如果你已经知道如何查阅文档 , 那么可以使用pycharm ;
2. 如果不知道怎么查阅文档 , 先熟悉qt设计师 , 然后选择  eric6 和 pycharm同时使用 , 因为eric6的智能提 
   示有点蛋疼 , 我也没完全搞清楚怎么配置 .

**Comment by 892768447** [comment]
没试过Eric6，相比Pycharm，LiClipse对PyQt5的提示要好很多

**Comment by mcz574** [comment]
vscode不好吗？
